From 62b0f9006c9553cf36ad9dc3cbec46a8f7fe82d2 Mon Sep 17 00:00:00 2001
From: Simon <simonzigon15@gmail.com>
Date: Sat, 24 Jan 2026 11:00:27 +1030
Subject: [PATCH] Include fix

---
 include/linux/rfnm-shared.h | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/include/linux/rfnm-shared.h b/include/linux/rfnm-shared.h
index dc3bf4f58e41..e30aba715ec7 100644
--- a/include/linux/rfnm-shared.h
+++ b/include/linux/rfnm-shared.h
@@ -92,14 +92,7 @@ struct __attribute__((__packed__)) rfnm_bootconfig {
 
 
 
-
-
-
-
-
-
-
-#include </home/davide/imx-rfnm-bsp/build/tmp/work-shared/imx8mp-rfnm/kernel-source/include/linux/rfnm-api.h>
+#include <linux/rfnm-api.h>
 
 
 
-- 
2.51.0

